### PR TITLE
Add regression tests of BA issue cases.

### PR DESCRIPTION
**Pull request by yzha107**

### CODE REVIEW

**Comment by yamt** [comment]
what's "BA issue"?

**Comment by yzha107** [comment]
> what's "BA issue"?

The cases that verify the issue fix of bytecode alliance wamr repo (this repo).

**Comment by TianlongLiang** [comment]
LGTM

File: tests/regression/ba-issues/README.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+```dummy
+=======================================================
+Failing issue id: 2945.
+run with command_lists: ['./build/iwasm-default-wasi-disabled', '--heap-size=0', '-f', 'to_test', '/home/tl/TL/clion_projects/wamr-internal-test/tests/regression/ba-issues/issues/issue-2945/iwasm_fast_interp_moob_unhandled.wasm']
+exit code (actual, expected) : (1, 0)
+stdout (actual, expected) : ('Exception: out of bounds memory access', 'Exception: out of bounds memory access')
+=======================================================
+```
+
+TODO: only run part of test cases?
```
Comment (wenyongh) [inline tests/regression/ba-issues/README.md:250]: How about removing the this line? We may implement it in the future but seems no need to add it in the document?

Comment (yzha107) [reply tests/regression/ba-issues/README.md:250]: OK. I also think it is better to remove it.

**Review by wenyongh — APPROVED** [review]
LGTM

**Pull request merged**